### DISCUSSION TITLE
Galaxy Brain

### DISCUSSION THREAD

**Discussion by MahdiAbbasi7** [community/community#72588] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Does anyone know why my galaxy brain budget isn't showing in my profile, but I got this budget last month?
this is my profile : 
![Screenshot from 2023-10-26 09-06-41](https://github.com/community/community/assets/77788727/ccb13a0c-277b-4acc-94c1-be6e4de100fc)
and this is my all budgets:
![Screenshot from 2023-10-26 09-07-53](https://github.com/community/community/assets/77788727/8feec2d1-9822-4273-b17f-b54b92ba3414)

**Comment by grandEL-dsouza** [comment]
The answers you gave might have been unaccepted. 

I would suggest you read this blog for more details:
https://github.com/4xmen/Get-Github-Achievements/blob/main/assets/steps/README.galaxy-brain.md

**Comment by ktauchathuranga** [comment, marked as answer]
in this case i think what happened is you accidently hide the badge from profile,

click on one of the badges shown in your profile
and then u will be redirected to your total badge page,
in there click on your Galaxy Brain badge and make sure its not hidden.
look at the picture i attached, left top corner. 
![image](https://github.com/community/community/assets/79928743/b5289ac1-e581-4f98-ac39-80be44f7725a)

happy coding 🖤
